### PR TITLE
[5.6] Added functions morphsUnique & dropMorphsUnique

### PR DESCRIPTION
**Pull request by gushh**
Add a unique index in morphs function. Useful for morphOne relationships.

__Write:__
```php
    $table->morphs('profile');
    $table->unique(['profile_type', 'profile_id']);
```

__Equals to:__
```php
    $table->morphsUnique('profile');
```

### CODE REVIEW

File: src/Illuminate/Database/Schema/Blueprint.php [hunk 1]
```diff
@@ -219,7 +219,8 @@ public function addFluentCommands(Grammar $grammar)
                 $value = $column->{$attributeName};
 
                 $this->addCommand(
-                    $commandName, compact('value', 'column')
+                    $commandName,
+                    compact('value', 'column')
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Schema/Blueprint.php]: Thanks for the PR. Please revert the unrelated changes though, like this one, and the one at the bottom of the diff.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Illuminate/Database/Schema/Blueprint.php (hunk 1 above)
Comment (Gushh) [reply src/Illuminate/Database/Schema/Blueprint.php]: :+1: Ready!

**Comment by jmarcher** [comment]
Can you add some tests, please?

**Comment by staudenmeir** [comment]
Why does it create a unique *and* a "normal" index?

**Comment by Gushh** [comment]
@jmarcher added grammar tests :+1: 

@staudenmeir a noob error! :+1:

**Comment by taylorotwell** [comment]
Just make the columns yourself using the other migration functions.
